src/material-experimental/mdc-chips/chip.ts

```diff
@@ -339,12 +346,13 @@ export class MatChip extends _MatChipMixinBase implements AfterContentInit, Afte
   _listenToRemoveIconInteraction() {
     this.removeIcon.interaction
         .pipe(takeUntil(this._destroyed))
-        .subscribe((event) => {
+        .subscribe(event => {
           // The MDC chip foundation calls stopPropagation() for any trailing icon interaction
           // event, even ones it doesn't handle, so we want to avoid passing it keyboard events
-          // for which we have a custom handler.
-          if (this.disabled || (event instanceof KeyboardEvent &&
-            this.HANDLED_KEYS.indexOf(event.keyCode) !== -1)) {
+          // for which we have a custom handler. Note that we assert the type of the event using
+          // the `type`, because `instanceof KeyboardEvent` can throw during server-side rendering.
+          if (this.disabled || (event.type.indexOf('key') === 0 &&
```

Comment:
Can we use `startsWith` instead of `indexOf === 0`?

Reply:
I don't mind it either way, but people would have to polyfill `startsWith` since it's not supported everywhere.

Reply:
It looks like its used a couple places already. Microsoft is ending support for IE11 in a month anyways, at which point I think we plan to stop officially supporting it

Reply:
Done.